validation-test/Concurrency/issue70019.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -O -Xfrontend -disable-availability-checking %s -parse-as-library -module-name main -o %t/main
+// RUN: %target-codesign %t/main
```

Comment:
This is an overkill, it’s building an executable test where I think running typechecker is good enough.